packages/dashboard/src/components/grid/index.tsx

```diff
@@ -1,15 +1,15 @@
+import type { PointerEventHandler, ReactNode } from 'react';
 import React, { useEffect, useState } from 'react';
 import { useDrag, useDrop } from 'react-dnd';
 import { useKeyPress } from '~/hooks/useKeyPress';
+import type { Position } from '~/types';
 import { MouseClick } from '~/types';
 import { ItemTypes } from '../dragLayer/itemTypes';
 import { gestureable } from '../internalDashboard/gestures/determineTargetGestures';
 import { DASHBOARD_CONTAINER_ID, getDashboardPosition } from './getDashboardPosition';
-
+import { useMeasure } from 'react-use';
```

Comment:
It's possible for tree-shaking to work with named imports, if the lib is setup correctly, which I am not sure. If not, you may want to change the syntax to import the hook directly.

https://github.com/streamich/react-use/blob/master/docs/Usage.md